packages/js/uri-resolver-extensions/src/ExtendableUriResolver.ts

```diff
@@ -23,31 +23,32 @@ export class ExtendableUriResolver extends UriResolverAggregatorBase<
   Error
 > /* $ */ {
   // $start: ExtendableUriResolver-extInterfaceUri-static
-  /** The default interface uri to which implementations should be registered */
-  public static extInterfaceUri: Uri = new Uri(
-    "wrap://ens/uri-resolver.core.polywrap.eth"
-  );
+  /** The supported interface uris to which resolver-ext implementations should be registered */
+  public static extInterfaceUris: Uri[] = [
+    Uri.from("wrap://ens/wraps.eth:uri-resolver-ext@1.1.0"),
+    Uri.from("wrap://ens/wraps.eth:uri-resolver-ext@1.0.0"),
+  ];
   // $end
 
   // $start: ExtendableUriResolver-extInterfaceUri
-  /** The active interface uri to which implementations should be registered */
-  public readonly extInterfaceUri: Uri;
+  /** The active interface uris to which implementations should be registered */
+  public readonly extInterfaceUris: Uri[];
   // $end
   private readonly _resolverName: string;
 
   // $start: ExtendableUriResolver-constructor
   /**
    * Create an ExtendableUriResolver
    *
-   * @param extInterfaceUri - URI Resolver Interface URI
+   * @param extInterfaceUris - URI Resolver Interface URIs
    * @param resolverName - Name to use in resolution history output
    * */
   constructor(
-    extInterfaceUri: Uri = ExtendableUriResolver.extInterfaceUri,
+    extInterfaceUris: Uri[] = ExtendableUriResolver.extInterfaceUris,
```

Comment:
nit: seeing that we have two variables with the same name (one static variable and the other instance variable) feels weird - should we define the static one as the default (explicitly adding something like `defaultExtInterfaceUris`)? iiuc, it's only used as a default value in the constructor

Reply:
There was a minor modification to the interface that was made, which caused the version to be bumped to `1.1.0`.

The change was that each method was given an optional environment, see here: https://wrappers.io/v/ipfs/QmSAXrSLcmGUkQRrApAtG5qTPmuRMMX2Zf1wihpguDQfbm

This was discussed up here: https://github.com/polywrap/toolchain/pull/1582#discussion_r1121424976

So we have to account for old & new interface URIs that wrappers could have implemented, since both will work as there are no breaking changes being made.

Reply:
Effectively it's just backwards compatibility.